org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/telemetry/TelemetryManager.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lemminx.telemetry;
+
+import org.eclipse.lsp4j.InitializedParams;
+import org.eclipse.lsp4j.services.LanguageClient;
+
+/**
+ * Telemetry manager.
+ *
+ * @author Angelo ZERR
+ */
+public class TelemetryManager {
+
+	/**
+	 * "startup" telemetry event name
+	 */
+	private static final String STARTUP_EVENT_NAME = "startup";
+
+	/**
+	 * "service" telemetry event name
+	 */
+	private static final String SERVICE_EVENT_NAME = "service";
+
+	private final LanguageClient languageClient;
+
+	private boolean enabled;
+
+	public TelemetryManager(LanguageClient languageClient) {
+		this.languageClient = languageClient;
+	}
+
+	public boolean isEnabled() {
+		return enabled;
+	}
+
+	public void setEnabled(boolean enabled) {
+		this.enabled = enabled;
+	}
+
+	/**
+	 * Send a telemetry event on start of the XML server
+	 *
+	 * @param params
+	 */
+	public void onInitialized(InitializedParams params) {
+		if (isEnabled()) {
+			telemetryEvent(STARTUP_EVENT_NAME, new InitializationTelemetryInfo());
+		}
+	}
+
+	/**
+	 * Send a telemetry event on call of a service.
+	 *
+	 * @param serviceName service name.
+	 */
+	public void onServiceCall(String serviceName) {
```

Comment:
seems unused

Reply:
Yes, this is left over from the old PR. I'll remove it.